### PR TITLE
[docs] Improve information about manually uploading images in a private environment

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Improved information about manually uploading images in a private environment.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix
summary: Improve information about manually uploading images in a private environment.
impact_level: low
```

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 1]
```diff
@@ -567,14 +567,14 @@ Deckhouse поддерживает работу только с Bearer token-с
...
    > Перед запуском команды убедитесь, что этот путь существует и у используемой учетной записи есть права на запись.
 
    Если ваш registry не требует авторизации, флаги `--registry-login`/`--registry-password` указывать не нужно.
 
-1. После загрузки образов в изолированный registry, отредактируйте YAML-манифест `ModuleSource`:
+1. После загрузки образов в изолированный registry отредактируйте YAML-манифест `ModuleSource`:
 
-   * Измените поле `.spec.registry.repo` на адрес, который вы указали в параметре `--registry` при выгрузке образов;
+   * измените поле `.spec.registry.repo` на адрес, который вы указали в параметре `--registry` при выгрузке образов;
    * Измените поле `.spec.registry.dockerCfg` на Base64-строку с данными для авторизации в вашем registry в формате `dockercfg`. Обратитесь к документации вашего registry для получения информации о том, как сгенерировать этот токен.
```
Comment (VV-corrector) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:578]: «Измените» с маленькой буквы

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 2]
```diff
@@ -419,14 +419,15 @@ This feature is only available in Standard Edition (SE), Enterprise Edition (EE)
...
+   For example, for Deckhouse 1.60, only one version, `1.60.4`, will be pulled, as it is sufficient to update Deckhouse from version `1.59`.
 
    The command below will pull Deckhouse tarballs for versions that are on the release channels (check [flow.deckhouse.io](https://flow.deckhouse.io) for the current status of the release channels):
 
    ```shell
-   d8 mirror pull --license "<DECKHOUSE_LICENSE_KEY>" $(pwd)/d8.tar
+   d8 mirror pull --source="registry.deckhouse.io/deckhouse/ee" --license="<DECKHOUSE_LICENSE_KEY>" $(pwd)/d8.tar
    ```
-
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:429]:
```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se`.
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 3]
```diff
@@ -437,9 +438,10 @@ This feature is only available in Standard Edition (SE), Enterprise Edition (EE)
    For example, here is how you can pull all Deckhouse version images starting from version 1.45:
 
    ```shell
-   d8 mirror pull --license="<DECKHOUSE_LICENSE_KEY>" --min-version=1.45 $(pwd)/d8.tar
+   d8 mirror pull --source="registry.deckhouse.io/deckhouse/ee" --license="<DECKHOUSE_LICENSE_KEY>" --min-version=1.45 $(pwd)/d8.tar
    ```
-
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:443]:
```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se`.
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 4]
```diff
@@ -479,7 +481,8 @@ This feature is only available in Standard Edition (SE), Enterprise Edition (EE)
    ```shell
    d8 mirror push /tmp/d8-images/d8.tar "your.private.registry.com:5000/deckhouse/ee" --registry-login="<USER>" --registry-password="<PASSWORD>"
    ```
-
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените последнюю часть ссылки на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `your.private.registry.com:5000/deckhouse/se`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:484]:
```suggestion
   > Make sure to edit the path to match your platform edition — replace the last section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `your.private.registry.com:5000/deckhouse/se`.
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 5]
```diff
@@ -503,22 +506,50 @@ The steps below are necessary for manually loading images of modules connected f
...
 1. [Download and install the Deckhouse CLI tool](https://github.com/deckhouse/deckhouse-cli/blob/main/README.md#how-to-install).
 
-   1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the command `d8 mirror modules pull`.
-
-      `d8 mirror modules pull` pulls only the module versions available in the module release channels at the time of copying unless the `--modules-filter` flag is set.
-
-      The following command will pull module images from the source described in the `ModuleSource` resource located in the `$HOME/module_source.yml` file:
+1. Создайте строку аутентификации для `registry.deckhouse.io` командой:
+   ```shell     
+   YOUR_LICENSE_KEY="ваш_лицензионный_ключ" base64 -w0 <<EOF
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:511]:
```suggestion
1. Create an authentication string for `registry.deckhouse.io` using the following command:
   ```shell     
   YOUR_LICENSE_KEY="your_license_key" base64 -w0 <<EOF
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 6]
```diff
@@ -503,22 +506,50 @@ The steps below are necessary for manually loading images of modules connected f
...
+         "registry.deckhouse.io": {
+           "auth": "$(echo -n license-token:${YOUR_LICENSE_KEY} | base64 -w0)"
+         }
+       }
+     }
+   EOF
+   ```
+   > Не забудьте заменить `ваш_лицензионный_ключ` на свой реальный лицензионный ключ.
+
+1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the command `d8 mirror modules pull`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:523]:
```suggestion
   > Make sure to replace the `your_license_key` placeholder with your actual license key.

1. Pull module images from their source registry, defined as a `ModuleSource` resource, into a dedicated directory using the `d8 mirror modules pull` command.
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 7]
```diff
@@ -503,22 +506,50 @@ The steps below are necessary for manually loading images of modules connected f
...
+
+   `d8 mirror modules pull` pulls only the module versions available in the module release channels at the time of copying unless the `--modules-filter` flag is set.
+   Пример файла `module_source.yml` с описанием ModuleSource:
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:526]:
```suggestion
   Below is an example of a `module_source.yml` file with the ModuleSource definition:
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 8]
```diff
@@ -503,22 +506,50 @@ The steps below are necessary for manually loading images of modules connected f
...
+   ```yaml
+   apiVersion: deckhouse.io/v1alpha1
+   kind: ModuleSource
+   metadata:
+     name: deckhouse
+   spec:
+     registry:
+       # Укажите строку, полученную в п.2 вместо CHANGE
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:534]:
```suggestion
       # Replace CHANGE with the string produced in step 2
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 9]
```diff
@@ -503,22 +506,50 @@ The steps below are necessary for manually loading images of modules connected f
...
+       dockerCfg: CHANGE
+       repo: registry.deckhouse.io/deckhouse/ee/modules
+       scheme: HTTPS
+     # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:538]:
```suggestion
     # Select the appropriate release channel: Alpha, Beta, EarlyAccess, Stable, or RockSolid
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 10]
```diff
@@ -540,6 +571,24 @@ The steps below are necessary for manually loading images of modules connected f
 
     * Change the `.spec.registry.repo` field to the address that you specified in the `--registry` parameter when you uploaded the images;
     * Change the `.spec.registry.dockerCfg` field to a base64 string with the authorization data for your registry in `dockercfg` format. Refer to your registry's documentation for information on how to obtain this token.
+ 
+   Пример:
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:575]:
```suggestion
   An example:
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 11]
```diff
@@ -540,6 +571,24 @@ The steps below are necessary for manually loading images of modules connected f
...
+   ```yaml
+   apiVersion: deckhouse.io/v1alpha1
+   kind: ModuleSource
+   metadata:
+     name: deckhouse
+   spec:
+     registry:
+       # Укажите строку, полученную в п.2 вместо CHANGE
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:583]:
```suggestion
       # Replace CHANGE with the string produced in step 2
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 12]
```diff
@@ -540,6 +571,24 @@ The steps below are necessary for manually loading images of modules connected f
...
+       dockerCfg: CHANGE
+       repo: registry.deckhouse.io/deckhouse/ee/modules
+       scheme: HTTPS
+     # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:587]:
```suggestion
     # Select the appropriate release channel: Alpha, Beta, EarlyAccess, Stable, or RockSolid
```

File: docs/documentation/pages/DECKHOUSE-FAQ.md [hunk 13]
```diff
@@ -540,6 +571,24 @@ The steps below are necessary for manually loading images of modules connected f
...
+     releaseChannel: "Stable"
+   ```
+
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы – замените в ссылке `ee` на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.io/deckhouse/se`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ.md:591]:
```suggestion
   > Make sure to edit the path to match your platform edition — replace the `ee` section of the link with `se` or `cse`. For example, for the SE edition, the link would look as follows: `registry.deckhouse.io/deckhouse/se/modules`.
```

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 14]
```diff
@@ -532,15 +567,33 @@ Deckhouse поддерживает работу только с Bearer token-с
...
+     registry:
+       # Укажите строку, полученную в п. 2 вместо CHANGE
+       dockerCfg: CHANGE
+       repo: registry.deckhouse.ru/deckhouse/ee/modules
+       scheme: HTTPS
+     # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid
+     releaseChannel: "Stable"
+   ```
+
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы — замените в ссылке `ee` на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.ru/deckhouse/se`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:596]: М.б. здесь ссылка должна быть `registry.deckhouse.ru/deckhouse/ee/modules` ? (если судить по полю `repo` выше)

**Review by kirkonru — APPROVED** [review]

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md (hunk 14 above)
Comment (Zhbert) [reply docs/documentation/pages/DECKHOUSE-FAQ_RU.md:596]: Да, спасибо.

File: docs/documentation/pages/DECKHOUSE-FAQ_RU.md [hunk 15]
```diff
@@ -532,15 +567,33 @@ Deckhouse поддерживает работу только с Bearer token-с
...
+     registry:
+       # Укажите строку, полученную в п. 2 вместо CHANGE
+       dockerCfg: CHANGE
+       repo: registry.deckhouse.ru/deckhouse/ee/modules
+       scheme: HTTPS
+     # Выберите подходящий канал обновлений: Alpha, Beta, EarlyAccess, Stable, RockSolid
+     releaseChannel: "Stable"
+   ```
+
+   > Не забудьте изменить путь на соответствующий вашей редакции платформы — замените в ссылке `ee` на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.ru/deckhouse/ee/modules`.
```
Comment (kirkonru) [inline docs/documentation/pages/DECKHOUSE-FAQ_RU.md:596]:
```suggestion
   > Не забудьте изменить путь на соответствующий вашей редакции платформы — замените в ссылке `ee` на `se` или `cse`. Например, для редакции SE ссылка будет выглядеть так: `registry.deckhouse.ru/deckhouse/se/modules`.
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8833](https://github.com/deckhouse/deckhouse/pull/8833) to the branch [release-1.61](https://github.com/deckhouse/deckhouse/tree/release-1.61) successful!

**Pull request merged**